server/meshmodel/argo-cd/relationships/sibling_matchlabels.json

```diff
@@ -0,0 +1,62 @@
+{
+  "apiVersion": "core.meshery.io/v1alpha1",
+  "kind": "Sibling",
+  "metadata": {
+    "description": "A relationship between two objects that are siblings."
```

Comment:
@aniskhalfallah Kindly refer to https://github.com/meshery/meshery/tree/master/server/meshmodel/kubernetes/relationships for adding descriptions for any kind of Relationship. In case you want to add an example in the description, you may use any example for the current model on which you are defining relationships